src/main/resources/testchipip/csrc/testchip_tsi.h

```diff
@@ -32,6 +38,6 @@ class testchip_tsi_t : public tsi_t
 
   bool is_loadmem;
   bool write_hart0_msip;
-  std::vector<std::pair<uint64_t, uint32_t>> init_writes;
+  std::vector<init_access_t> init_accesses;
```

Comment:
Does this mean that we can have as many `+init_write`s and `+init_read`s as we want before the binary?

Reply:
Yes